contracts/test/PoolFDTLosses.t.sol

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.11;
+pragma experimental ABIEncoderV2;
+
+import "./TestUtil.sol";
+
+contract PoolClaimTest is TestUtil {
+
+    using SafeMath for uint256;
+
+    function setUp() public {
+        setUpGlobals();
+        setUpTokens();
+        setUpOracles();
+        setUpFactories();
+        setUpCalcs();
+        setUpActors();
+        setUpBalancerPool();
+        setUpLiquidityPools();
+        createLoans();
+    }
+
+    function getSpecs(
+        uint256 apr,
+        uint256 index,
+        uint256 numPayments,
+        uint256 requestAmount,
+        uint256 collateralRatio,
+        uint256 minimumRequestAmt
+    ) internal pure returns (uint256[5] memory) {
+        // Adding extra 25% to make sure pool get losses.
+        minimumRequestAmt = (125 * minimumRequestAmt) / 100;
+        uint256 maxRequestAmt = 1E7 * USD;
+
+        return getFuzzedSpecs(apr, index, numPayments, requestAmount, collateralRatio, minimumRequestAmt, 0, maxRequestAmt);
+    }
+
+    function getLoanAndDepositAmount(
```

Comment:
`createLoanAndGetDepositAmount`?

Reply:
I also think maybe taking out the depositAmount logic from this function and just calling it `createLoanForLosses` might make more sense

Reply:
The issue is I can't afford the reference to `specs` as a stack variable.